backend/src/common/types/common/common.ts

```diff
@@ -0,0 +1 @@
+export { IdContainer } from './id-container//id-conteinter.type';
```

Comment:
I also thought in that way, but we may have same cases in future with name container, or some text container, etc. and I think they should be logically separated from other types because their purpose for me is more common (work with ids not binded to any specific type)